src/transformers/integrations.py

```diff
@@ -55,7 +55,10 @@
 
 _has_neptune = importlib.util.find_spec("neptune") is not None
 if TYPE_CHECKING and _has_neptune:
-    from neptune.new.metadata_containers.run import Run
+    try:
+        from neptune import Run
+    except ImportError:
+        from neptune.new import Run
```

Comment:
Could we test on the version instead for the imports? You can look at [here](https://github.com/huggingface/transformers/blob/00934026a4832f030bb37131968668c7db3ca7cf/src/transformers/utils/import_utils.py#L52) to see how to get the version without importing the package.

Reply:
Hi @sgugger ! Thank you for the comment. I've just pushed a commit that I believe implements your suggestions. Would you let me know if it really is in the spirit of what you proposed?